README.md

```diff
@@ -109,7 +109,7 @@ When passing additional parameters to GET based request use the following syntax
  client.contents('octokit/octokit.rb', path: 'path/to/file.rb', query: {ref: 'some-other-branch'})
 ```
 
-[API methods]: http://octokit.github.io/octokit.rb/method_list.html
+[api methods]: http://octokit.github.io/octokit.rb/method_list.html
```

Comment:
nit: API should be in caps given it is an acronym - definitely not a show stopper for the great changes in this PR though

```suggestion
[API methods]: http://octokit.github.io/octokit.rb/method_list.html
```